.toolkit/plugins/commitlint/.toolkitrc.yml

```diff
@@ -1,3 +1,7 @@
-hooks:
-  git:commitmsg:
-    - Commitlint
+tasks:
+  Commitlint: './index.js'
```

Comment:
**question:** @ivomurrell is it possible to extend this plugin to run on CI?

Reply:
It's possible yeah, [here's the command](https://github.com/rowanmanning/actions/blob/95bd5a6df32e7304563a46513e0efa026ea91366/.github/workflows/enforce-commit-style.yml#L21-L24)

Reply:
yes! if they wanted it to run in CI they could set the task to run in the `test:ci`